content/articles/vue-data.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Delivering Data to a Vue Component"
+date: 2019-05-05
+type: "post"
+authors: ["dreher"]
+tags: ["django","vue","python","javascript","json"]
+lede: "Delivering data from server-side to client-side is pretty
+straightforward. Generally, you just throw together an API, RESTful or
+otherwise. When working in the Django framework, there are a couple of other
+options to consider, that might be easier and solve a few problems."
+shortlede: "Exploring efficient approaches to get data clientside with Django
+and Vue."
+poster: "poster-vue-data.jpg"
+socmediaimg: "socmediaimg-vue-data.jpg"
+hiliteimg: "poster-vue-data.jpg"
+poster_sourceurl: "https://unsplash.com/photos/fb7yNPbT0l8"
+poster_source: "Photo by Mathyas Kurmann on Unsplash"
+bookendanimal: "spider"
+---
+
+## Special delivery
+
+Delivering data from server to client is pretty straightforward. Generally, you
+just throw together an API, RESTful or otherwise. When working in the Django
+framework, there are a couple of other options to consider, that might be
+easier and solve a few problems. Here are two ways I've used to get data to the
```

Comment:
Unclear: `that might be easier and solve a few problems.` Is this referring to the options?